### ISSUE TITLE
Tuneup suddenly dislikes pedantic

### ISSUE THREAD

**Issue by domesticmouse** [google/tuneup.dart#89]
Hey @devoncarew,

For some reason, I'm hitting an issue with `tuneup` taking a sudden dislike to `pedantic`:

```
analyze
  /Users/brettmorgan/homebrew/Cellar/dart/2.10.4/libexec/bin/pub run tuneup check
  Checking project dart_services...
  
    warning - The include file package:pedantic/analysis_options.yaml in /Users/brettmorgan/Documents/GitHub/dart-services/analysis_options.yaml cannot be found at analysis_options.yaml:1:11 - (include_file_not_found)
  
  1 issue found; analyzed 109 source files in 9.0s.

failed with exit code 1
#0      run (package:grinder/src/run.dart:48:5)
#1      Pub.run (package:grinder/grinder_sdk.dart:241:12)
#2      analyze (file:///Users/brettmorgan/Documents/GitHub/dart-services/tool/grind.dart:26:7)
#3      _LibraryMirror._invoke (dart:mirrors-patch/mirrors_impl.dart:1089:36)
#4      _ObjectMirror.invoke (dart:mirrors-patch/mirrors_impl.dart:242:25)
#5      TaskDiscovery.discoverDeclaration.<anon> (package:grinder/src/discover_tasks.dart:77:38)
#6      _rootRun (dart:async/zone.dart:1190:13)
#7      _CustomZone.run (dart:async/zone.dart:1093:19)
#8      _runZoned (dart:async/zone.dart:1630:10)
#9      runZoned (dart:async/zone.dart:1550:10)
#10     ZonedValue.withValue (package:grinder/src/utils.dart:113:12)
#11     GrinderTask.execute (package:grinder/src/grinder_task.dart:63:27)
#12     Grinder._invokeTask (package:grinder/src/grinder.dart:193:27)
#13     Grinder.start.<anon> (package:grinder/src/grinder.dart:168:16)
#14     Future.forEach.<anon> (dart:async/future.dart:512:26)
#15     Future.doWhile.<anon> (dart:async/future.dart:554:26)
#16     _RootZone.runUnaryGuarded (dart:async/zone.dart:1384:10)
#17     _RootZone.bindUnaryCallbackGuarded.<anon> (dart:async/zone.dart:1423:26)
#18     _RootZone.runUnary (dart:async/zone.dart:1450:54)
#19     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#20     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#21     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#22     Future._completeWithValue (dart:async/future_impl.dart:529:5)
#23     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
#24     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
#25     generateProtos (file:///Users/brettmorgan/Documents/GitHub/dart-services/tool/grind.dart)
```

Any ideas on how to debug this?

brett

**Comment by devoncarew** [comment]
I'm not sure - do all the analysis_options files look reasonable?

You may also try the `dart analyze` command. It's a replacement for `tuneup`, and is available starting from Dart 2.10.0 and on.

**Comment by domesticmouse** [comment]
The [analysis_options.yaml](https://github.com/domesticmouse/dart-services/blob/flutter-1.24-attempt-2/analysis_options.yaml) hasn't been touched in 8 months, so I doubt it's at issue. Updating to `dart analyze` worked nicely tho. =)
